docs/APIReference-ContentState.md

```diff
@@ -170,36 +170,39 @@ objects.
 
 ## Static Methods
 
-### createFromText
```

Comment:
Unlike previous headings e.g. `getDepth()`, this and later method names are missing parens `()`. While you're at it, do you mind adding them here and in the other occurrences? Thanks!

Reply:
I wasn't sure whether the omission of the parens was intentional so I left them out. Have added them to all the functions/methods in the API section!